static/bidder-info/triplelift_native.yaml

```diff
@@ -9,7 +9,6 @@ capabilities:
     mediaTypes:
       - native
 userSync:
-  key: "triplelift"
```

Comment:
Why was the key being dropped?

Reply:
I'd like to keep the identity pools exactly as they are right now. I'll put this in a separate PR for review from TripleLift at a later date.